### DISCUSSION TITLE
v3.0.0

### DISCUSSION THREAD

**Discussion by SKProCH** [AvaloniaCommunity/Material.Avalonia#289] [Announcements]
## What's Changed
* Support for Avalonia 11
* Implement new MaterialTheme - preferred way to import styles
* All styles migrated to ControlThemes to avoid styles leaking
* Brushes names changed to make it more consistent
* Dialogs and DataGrid extracted to separate packages
* Implement styles for many controls, e.g. TimePicker
* Many other changes

## New Contributors
* @OsaPL made their first contribution in https://github.com/AvaloniaCommunity/Material.Avalonia/pull/130
* @MarioGK made their first contribution in https://github.com/AvaloniaCommunity/Material.Avalonia/pull/136
* @kirichenec made their first contribution in https://github.com/AvaloniaCommunity/Material.Avalonia/pull/140
* @tpetruk made their first contribution in https://github.com/AvaloniaCommunity/Material.Avalonia/pull/143
* @dojo90 made their first contribution in https://github.com/AvaloniaCommunity/Material.Avalonia/pull/147
* @dylanlangston made their first contribution in https://github.com/AvaloniaCommunity/Material.Avalonia/pull/150
* @Mrxx99 made their first contribution in https://github.com/AvaloniaCommunity/Material.Avalonia/pull/194
* @graealex made their first contribution in https://github.com/AvaloniaCommunity/Material.Avalonia/pull/218
* @JanTamis made their first contribution in https://github.com/AvaloniaCommunity/Material.Avalonia/pull/239
* @rabbitism made their first contribution in https://github.com/AvaloniaCommunity/Material.Avalonia/pull/256
* @Janek91 made their first contribution in https://github.com/AvaloniaCommunity/Material.Avalonia/pull/270
* @Lehonti made their first contribution in https://github.com/AvaloniaCommunity/Material.Avalonia/pull/276

**Full Changelog**: https://github.com/AvaloniaCommunity/Material.Avalonia/compare/v2.5.1...v3.0.0

<hr /><em>This discussion was created from the release <a href='https://github.com/AvaloniaCommunity/Material.Avalonia/releases/tag/v3.0.0'>v3.0.0</a>.</em>

**Comment by anna-dolbina** [comment]
Tried updating to this release with Avalonia 11.0.4 and got an exception during an attempt to display the context menu:
"ControlTemplate styles cannot contain multiple template selectors".

Everything works properly with `3.0.0-avalonia11-preview5.203-nightly` and seems to break with `preview6`.
Is this some kind of a known issue?

**Reply by SKProCH** [reply]
Hello. Can you share your project? 
Or did you set up a custom context menu or something like that? If yes, please share code.

**Reply by SKProCH** [reply]
@anna-dolbina seems like fixed in https://github.com/AvaloniaCommunity/Material.Avalonia/issues/290
Can you try the latest nightly?

**Reply by anna-dolbina** [reply]
@SKProCH the exception seems to be gone in `3.0.0.35-nightly`. Thanks!
